apps/web/cypress/tests/auth.spec.ts

```diff
@@ -238,9 +241,7 @@ describe('User Sign-up and Login', function () {
       const date = new Date(Date.now() + THIRTY_DAYS + ONE_MINUTE);
       cy.clock(date);
 
-      cy.visit('/subscribers');
-
-      cy.waitLoadFeatureFlags();
+      cy.getByTestId('side-nav-subscribers-link').click();
```

Comment:
when using the `cy.visit('/subscribers');` it looks like that application is hanging with the loader... @antonjoel82 we have to verify that when the token is expired and the user refreshes the page he will be navigated to the login page...